expression_json.go

```diff
@@ -31,71 +31,77 @@ func parseJsonField(data []interface{}) (Expression, error) {
 	return out, nil
 }
 
-func parseJsonEquals(data []interface{}) (Expression, error) {
+func parseJsonComparison(data []interface{}) (Expression, Expression, error) {
 	lhsData, ok := data[1].([]interface{})
 	if !ok {
-		return nil, errors.New("invalid equals expression lhs format")
+		return nil, nil, errors.New("invalid equals expression lhs format")
```

Comment:
this function is used by more than just equals (i.e. not equals). So this error may not reflect the accurate caller.

Reply:
Good call.  I actually had to fix this exact problem in the core expression side earlier today, should have caught it!  Will fix it shortly.

Reply:
Done.